### PR TITLE
Adds beaded curtain and some jewelry

### PR DESCRIPTION
**Pull request by curstwist**
#### Summary

SUMMARY: Content "Adds a beaded curtain doorway, beaded bracelet, beaded necklace & wooden beads"

#### Purpose of change
 Adds variety for doorways.

#### Describe the solution

Made a beaded curtain door that is rather noisy to move through.   It can be constructed & deconstructed.  I also added some wood bead jewelry to give the component an added use.  The jewelry can be inscribed since it is wood.

#### Describe alternatives you've considered

Making it free standing like the door curtain but thought keeping it to door frames made more sense.

#### Additional context
sample friendship bracelets/necklaces for NPCs 
lable
inscription

### CODE REVIEW

**Comment by Mecares** [comment]
Maybe give those beads stats that makes them usable as sling ammo?

**Comment by curstwist** [comment]
> 
> 
> Maybe give those beads stats that makes them usable as sling ammo?

I considered this but didn't have a good idea on what sort of damage stats they could get.  If someone more familiar with the damage/ammo numbers would like to post reasonable values, I can add it as slingshot ammo.

**Comment by Fris0uman** [comment]
Shouldn't the jewelry get the `fancy` tag ?

**Comment by curstwist** [comment]
> 
> 
> Shouldn't the jewelry get the `fancy` tag ?

I woudn't think so, they are just some wooden beads & string but open to changing it if others agree.  Do all accessories get that tag?  What would be cool is if giving them out to NPC's increased NPC trust/friendship.

**Comment by Fris0uman** [comment]
> Do all accessories get that tag?

I'm not sur but I think hairpin has it and it's just an hairpin...

>giving them out to NPC's increased NPC trust

A `frienship` flag ! YES ! X)

**Comment by curstwist** [comment]
> 
> 
> > Do all accessories get that tag?
> 
> I'm not sur but I think hairpin has it and it's just an hairpin...
> 
> > giving them out to NPC's increased NPC trust
> 
> A `frienship` flag ! YES ! X)

took a quick look, that tag is used oddly.  I'll add it but I think a review of the use of both fancy & super_fancy should be done at some point.  e.g. copper bracelet gets neither, diamond ring is only fancy, crowns are super_fancy though.

**Comment by kevingranade** [comment]
I wouldn't bother making them suitable ammo, wood and glass make very poor
sling bullets.

File: data/json/recipes/armor/other.json [hunk 1]
```diff
@@ -95,6 +95,30 @@
     "type": "recipe",
     "components": [ [ [ "silver_small", 15 ] ] ]
   },
+  {
+    "result": "bead_bracelet",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_OTHER",
+    "skill_used": "fabrication",
+    "difficulty": 0,
```
Comment (ZhilkinSerg) [inline data/json/recipes/armor/other.json]:
```suggestion
```

File: data/json/recipes/armor/other.json [hunk 2]
```diff
@@ -95,6 +95,30 @@
...
+    "autolearn": true,
+    "type": "recipe",
+    "components": [ [ [ "wooden_bead", 10 ] ], [ [ "string_6", 1 ] ] ]
+  },
+  {
+    "result": "bead_necklace",
+    "category": "CC_ARMOR",
+    "subcategory": "CSC_ARMOR_OTHER",
+    "skill_used": "fabrication",
+    "difficulty": 0,
```
Comment (ZhilkinSerg) [inline data/json/recipes/armor/other.json]:
```suggestion
```

File: data/json/recipes/armor/other.json (hunk 1 above)
Comment (curstwist) [reply data/json/recipes/armor/other.json]: I was getting an error when I set difficulty to 0.  I can try again...

File: data/json/recipes/armor/other.json (hunk 2 above)
Comment (curstwist) [reply data/json/recipes/armor/other.json]: Load error in game when it was set to 0.

File: data/json/recipes/armor/other.json (hunk 1 above)
Comment (curstwist) [reply data/json/recipes/armor/other.json]: removed line

**Pull request merged**